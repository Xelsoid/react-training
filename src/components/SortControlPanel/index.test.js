import React from 'react';
import ComponentExample from './index';

describe('SortControlPanel test', () => {
  it('renders correctly', () => {
    const component = shallow(<ComponentExample />);
    expect(component).toMatchSnapshot();
  });
});
