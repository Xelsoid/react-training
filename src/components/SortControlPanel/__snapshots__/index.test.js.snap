// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortControlPanel test renders correctly with child component 1`] = `
<div
  className="sort-control-panel-wrapper"
>
  <strong
    className="sort-control-panel__result-amount"
  >
    Films by drama genre
  </strong>
  <div
    className="sort-control-panel__filter"
  >
    <span
      className="sort-control-panel__filter-title"
    >
      SORT BY
    </span>
    <OptionChooser
      defaultValue="title"
      onChangeCallback={[MockFunction]}
      optionsConfig={
        Array [
          Object {
            "name": "searchType",
            "radioBtnName": "TITLE",
            "value": "title",
          },
          Object {
            "name": "searchType",
            "radioBtnName": "GENRE",
            "value": "genres",
          },
        ]
      }
    />
  </div>
</div>
`;

exports[`SortControlPanel test renders correctly without child component 1`] = `
<div
  className="sort-control-panel-wrapper"
>
  <strong
    className="sort-control-panel__result-amount"
  >
    Films by drama genre
  </strong>
</div>
`;
